tagstudio/src/qt/ts_qt.py

```diff
@@ -4601,7 +4601,7 @@ def expand_collation(self, collation_entries: list[tuple[int, int]]):
 						 for x in collation_entries])
 		# self.update_thumbs()
 
-	def get_frame_contents(self, index=0, query=str):
+	def get_frame_contents(self, index=0, query: str = None):
```

Comment:
I assume `query=str` was supposed to be `query:str`, right? And since it's after a parameter with a default value, it needs some default value as well.

Reply:
Yes, that looks to be like the original intention. Surprised my linter didn't pick that up...